File: 10/book.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    FILE *book, *verbs, *ans;
    book = fopen("book.txt", "r");
    verbs = fopen("verbs.txt", "r");
    ans = fopen("ans.txt", "a");

    char word[30], s[30];
    while (!feof(book)) {
        fscanf(book, "%s", word);
        while (!feof(verbs)) {
            fscanf(verbs, "%s", s);
            if (strstr(word, s)) {
                fprintf(ans, word);
            }
        }
    }
    return 0;
}

File: Python/Scraper_and_book/book.py
import re

read_book()


def read_book():
    with open('book.txt', 'r') as b:
        for line in b:
            for word in line.split():
                check(word)


def check(word):
    with open('verbs.txt', 'r') as f:
        lines = f.readlines()
        for line in lines:
            with open('ans.txt', 'a') as ans:
                if word in line and not line.split()[0] in ans.txt:
                    words = line.split()
                    for i in range(6):
                        ans.write(words[i] + ' ')
                    print >> ans
